strings/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'com.android.library'
+}
+
+android {
+    compileSdkVersion 30
```

Comment:
Is there really no way to set all these values once across modules? It's going to be really easy to make a mistake and not update all.

Reply:
Yeah we can probably use the same mechanism we use for the Dagger version. I think you're right that this would be the time to do it.